changelogs/unreleased/7676-metrics-retention.yml

```diff
@@ -0,0 +1,6 @@
+description: Reduce metrics retention time to two weeks
+issue-nr: 7676
+change-type: minor
```

Comment:
If this goes into iso6, this should become patch instead.

Reply:
```suggestion
change-type: patch
```